cognee/modules/retrieval/utils/description_to_codepart_search.py

```
@@ -11,7 +13,8 @@
 from cognee.modules.search.methods import search
 from cognee.infrastructure.llm.get_llm_client import get_llm_client
 
-logger = logging.getLogger(__name__)
+setup_logging(logger.ERROR)
```

Ruff (0.8.2)

Undefined name `logger`

(F821)